helpers/data_processor.py

```diff
@@ -45,6 +46,22 @@ def preprocess_data(self):
         """
         pass
 
+    def _push_unique_count(self, df, column_name, xcom_key):
```

Comment:
You can make it a @staticmethod since it is not using the self.
And what about giving the column directly instead of the whole dataframe?
Like `_push_unique_count(df_egapro.siren, "nb_siren_egapro")`

Reply:
I updated your comment because tou tagged someone called staticmethod LOL

Reply:
Since it's a utility method I don't see harm in using a static method.
And good point for only the column as input :)

Reply:
Damn the guy must be drown under notifications
Sorry dear staticmethod if you happen to have come other here 😅